fake_filesystem_unittest_test.py

```diff
@@ -207,6 +207,29 @@ def testCopyRealFile(self):
             self.copyRealFile(real_file_path, fake_file_path,
                               create_missing_dirs=False)
 
+    def testCopyRealFileNoDestination(self):
```

Comment:
Oh yeah, i Just copied the existing test and changed the parameters on the method call.

OK, I'll trim all that out.